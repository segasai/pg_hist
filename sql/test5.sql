select * from pg_hist_1d('select a from test1',ARRAY[1], ARRAY[0],ARRAY[2,2]);
select * from pg_hist_1d('select a from test1',ARRAY[-1], ARRAY[0],ARRAY[2]);
select * from pg_hist_2d('select a,b from test1',ARRAY[100,100,100], ARRAY[-0.1,-0.1],ARRAY[99.9,99.9]);
select * from pg_hist_3d('select a,b from test1',ARRAY[100,100], ARRAY[-0.1,-0.1],ARRAY[99.9,99.9]);
