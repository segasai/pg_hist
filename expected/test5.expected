psql:sql/test5.sql:1: ERROR:  The length of all arrays must be the same
psql:sql/test5.sql:2: ERROR:  All the histogram dimensions must be > 0
psql:sql/test5.sql:3: ERROR:  The length of all arrays must be the same
psql:sql/test5.sql:4: ERROR:  The dimensionality mismatch
